graylog2-server/src/main/java/org/graylog/plugins/sidecar/rest/models/Collector.java

```diff
@@ -66,6 +101,31 @@ public abstract class Collector {
     @Nullable
     public abstract String defaultTemplate();
 
+    @JsonProperty(FIELD_DEFAULT_TEMPLATE_CRC)
+    @Nullable
+    public abstract Long defaultTemplateCRC();
+
+    @JsonIgnore
+    public boolean defaultTemplateUpdated() {
+        long crc = checksum(defaultTemplate().getBytes(StandardCharsets.UTF_8));
```

Comment:
`defaultTemplate()` is `@Nullable` so `getBytes` may produce `NullPointerException`. Don't know if this really can happen but I think we should handle this to be on the safe side.